core/src/main/scala/org/apache/spark/scheduler/DAGScheduler.scala

```diff
@@ -1162,17 +1190,20 @@ class DAGScheduler(
         }
 
       case commitDenied: TaskCommitDenied =>
-        // Do nothing here, left up to the TaskScheduler to decide how to handle denied commits
+        // Do nothing here, left up to the TaskScheduler implementation
+        // to decide how to handle denied commits
 
       case ExceptionFailure(className, description, stackTrace, fullStackTrace, metrics) =>
-        // Do nothing here, left up to the TaskScheduler to decide how to handle user failures
+        // Do nothing here, left up to the TaskScheduler implementation
+        // to decide how to handle user failures
```

Comment:
Why was this changed? it's not a big deal, but it was OK on one line. Changes always have this small but non-zero chance of tangling up a merge later, so trivial changes are usually avoided